packages/api/src/platforms/vtex/clients/search/index.ts

```diff
@@ -84,11 +87,23 @@ export const IntelligentSearch = (
   const products = (args: Omit<SearchArgs, 'type'>) =>
     search<ProductSearchResult>({ ...args, type: 'product_search' })
 
+  const suggestedProducts = (
+    args: Omit<SearchArgs, 'type'>
+  ): Promise<ProductSearchResult> =>
+    fetchAPI(`${base}/api/suggestion_products/?term=${args.query}`)
+
+  const suggestedTerms = (
+    args: Omit<SearchArgs, 'type'>
+  ): Promise<SuggestedTerms> =>
+    fetchAPI(`${base}/api/split/suggestion_search/?q=${args.query}`)
```

Comment:
What's the difference between the results from these routes and the results from `api/autocomplete`?